contracts/RMRK/equippable/RMRKEquippable.sol

```diff
@@ -204,18 +204,18 @@ contract RMRKEquippable is
 
     // ------------------------------- EQUIPPING ------------------------------
 
-    function unnestChild(
+    function _unnestChild(
         uint256 tokenId,
         uint256 index,
         address to,
         bool isPending
-    ) public virtual override onlyApprovedOrOwner(tokenId) {
+    ) internal virtual override {
```

Comment:
So right now all gating is on the implementations?

Reply:
Only collection owner gating is on implementations. (Add resource, Add resource to token)
For token owner, all the core contracts now have both the public gated version which calls the internal one. (Accept resources and children, equip, etc).